.github/workflows/reusable-change-detection.yml

```
@@ -142,9 +144,9 @@ jobs:
         format: csv  # works for paths with spaces
     - name: Check for changes in MSI installer-related files
       if: >-
-        steps.changed-win-msi-files.outputs.added_modified_renamed != ''
+        (github.event_name == 'pull_request'
+           && steps.changed-win-msi-files.outputs.added_modified_renamed != ''
+        ) || github.event_name == 'workflow_dispatch'
```

... or change this to match the docs one:
```suggestion
        (github.event_name == 'pull_request'
           && steps.changed-win-msi-files.outputs.added_modified_renamed != '')
        || github.event_name == 'workflow_dispatch'
```